### DISCUSSION TITLE
Unable to download symbol file on 22000.2416

### DISCUSSION THREAD

**Discussion by SheepMoods** [valinet/ExplorerPatcher#2163] [General]
I tried to install the latest version of Explorer Patcher (22621.2283.57.2) on my Windows 11 21H2, but after the installation was completed he was unable to download the corresponding symbol file, the new start menu and could not be opened.

**Comment by Amrsatrio** [comment]
I have experienced a similar issue to yours, one workaround would be to try another network such as your phone's mobile/personal hotspot. Their symbol servers are a pain to download from sometimes.

**Comment by pyrates999** [comment]
I would update to windows 11 22h2.

**Comment by Amrsatrio** [comment]
I would've stayed on 22000 myself. The amount of breaking stuff Microsoft kept pushing into 22621 is atrocious, so there's a reason for that. Plus, I've tried my latest EP changes on that particular build, and symbols were able to be downloaded correctly.
